lighthouse-core/gather/drivers/driver.js

```diff
@@ -80,11 +78,11 @@ class Driver {
    * @return {!Promise<null>}
    */
   connect() {
-    return Promise.reject(new Error('Not implemented'));
+    return this._connection.connect().then(_ => this.sendCommand('Runtime.enable'));
```

Comment:
should the `sendCommand('Runtime.enable')` be moved out into gather-runner.js? It's a little weird that it's in here...I assume it's only for historical reasons